scripts/scan-txlog/scan_txlog.py

```diff
@@ -4324,6 +4341,16 @@ async def main():
                 if len(missing_in_script) > 0:
                     missing = [found_in_snapshot[cid] for cid in missing_in_script]
                     LOG.error(f"Contracts missing in script ACS: {missing}")
+                if args.scan_balance_assertions:
+                    # this will only work if a snapshot was taken, which is guaranteed by compare_acs_with_snapshot=True
+                    token_metadata = await scan_client.get_amulet_token_metadata()
```

Comment:
This will only work once amulet-0.1.14 has been voted in correct? Consider gating the check by exposing whether that's the case in https://github.com/hyperledger-labs/splice/blob/c062908d98ab604ec4ab155a5c0c85733a78ec12/apps/scan/src/main/openapi/scan.yaml#L1520

Reply:
Hmm... probably better to not do that to avoid accidentally not testing the balance computation as part of the sanity check plugin. If needed we could add a separate parameter to disable this check in case it causes problems.